### PR TITLE
Minor misc fixes

### PR DESCRIPTION
**Pull request by PeterMis**
- Schedule name length validation
- Displays : filter by XMR registered status resolves https://github.com/xibosignage/xibo/issues/3079
- Commands : execute with selected Displays / Display Groups, resolves https://github.com/xibosignage/xibo/issues/2927
- Display Groups : Show Display name when XMR action fails, resolves https://github.com/xibosignage/xibo/issues/3080
- phpcs / formatting adjustments

### CODE REVIEW

File: lib/Service/PlayerActionService.php [hunk 1]
```diff
@@ -78,35 +77,56 @@ private function getConfig()
...
+        }
 
         // Send a message to all displays
         foreach ($displays as $display) {
             /* @var Display $display */
-            if ($display->xmrChannel == '' || $display->xmrPubKey == '')
-                throw new InvalidArgumentException(__('This Player is not configured or ready to receive push commands over XMR. Please contact your administrator.'), 'xmrRegistered');
+            if ($display->xmrChannel == '' || $display->xmrPubKey == '') {
+                throw new InvalidArgumentException(
+                    __(sprintf(
```
Comment (dasgarner) [inline lib/Service/PlayerActionService.php:109]: This needs to be the other way around `sprintf(__())`

File: lib/Service/PlayerActionService.php [hunk 2]
```diff
@@ -78,35 +77,56 @@ private function getConfig()
...
+            }
 
             $displayAction = clone $action;
 
             try {
                 $displayAction->setIdentity($display->xmrChannel, $display->xmrPubKey);
             } catch (\Exception $exception) {
-                throw new InvalidArgumentException(__('Invalid XMR registration'), 'xmrPubKey');
+                throw new InvalidArgumentException(
+                    __(sprintf(
```
Comment (dasgarner) [inline lib/Service/PlayerActionService.php:124]: As above

File: lib/Service/PlayerActionService.php [hunk 3]
```diff
@@ -144,14 +165,20 @@ public function processQueue()
...
                 $this->log->error('Player action connection failed. E = ' . $sockEx->getMessage());
                 $failures++;
             }
         }
 
-        if ($failures > 0)
-            throw new ConfigurationException(sprintf(__('%d of %d player actions failed'), $failures, count($this->actions)));
+        if ($failures > 0) {
+            throw new ConfigurationException(
+                sprintf(
```
Comment (dasgarner) [inline lib/Service/PlayerActionService.php:176]: This one is the right way around :smile:

**Review by dasgarner — CHANGES_REQUESTED** [review]
Translation tweaks in commented cases.

File: lib/Service/PlayerActionService.php (hunk 1 above)
Comment (PeterMis) [reply lib/Service/PlayerActionService.php]: whoops, fixed

**Review by dasgarner — APPROVED** [review]

**Pull request merged**